### PR TITLE
Implement UI for users to accept suggestions from ChatGPT

### PR DESCRIPTION
**Pull request by botirk38**
### Description:
This PR introduces a toggleable sidebar for displaying suggestions related to the resume sections. It also ensures that the section names in the sidebar have their first character capitalized. The sidebar can be toggled open and closed, and suggestions can be accepted to add them to the respective sections.

### Changes:
1. **Utility Function:**
   - Created `utils.js` with a `capitalizeFirstLetter` function to capitalize the first letter of a string.

2. **Sidebar Component:**
   - Created `Sidebar.js` to display suggestions in a sidebar.
   - Styled the sidebar with `Sidebar.css`.
   - Added a close button to hide the sidebar.

3. **App Component:**
   - Integrated the `Sidebar` component.
   - Added a button to toggle the sidebar.
   - Updated the suggestions handling to use the sidebar.

## Issue Reference

References Issue #6

### CODE REVIEW

**Comment by Ahmedsaed** [comment]
I believe it would be better if you moved the sidebar styles into their own file, `sidebar.css`. This is just my preference.

**Comment by botirk38** [comment]
> I believe it would be better if you moved the sidebar styles into their own file, `sidebar.css`. This is just my preference.

I agree will get this done now.

**Comment by Ahmedsaed** [comment]
Quick Note:

To properly link an issue to the PR, use one of the following
- close
- closes
- closed
- fix
- fixes
- fixed
- resolve
- resolves
- resolved

**Example**: 
Closes #6 

This way when the PR is merged the issue is going to be closed automatically (It's also going to be linked in the development section of the right sidebar)

File: src/App.css [hunk 1]
```diff
@@ -51,3 +53,49 @@ button {
...
+
+.suggestion button {
+  margin-left: 10px;
+}
+
+.open-sidebar-btn {
+  position: absolute;
+  bottom: 2%;
+  right: 2%;
+}
```
Comment (Ahmedsaed) [inline src/App.css:101]: Just move these styles into their own file

**Review by Ahmedsaed — CHANGES_REQUESTED** [review]
LGTM!

**Pull request merged**